js/chat_app/src/components/NewChatForm/NewChatForm.jsx

```diff
@@ -92,7 +92,7 @@ export const NewChatForm = ({ open, onClose }) => {
                     ))}
                 </Select>
               </FormControl>
-              <Button variant="outlined" onClick={submitHandler}>
+              <Button variant="outlined" onClick={submitHandler} onSubmit>
```

Comment:
From my previous comment, I meant skip onClick for Button and moved all login to onSubmit={submitHandler} for the form tag.